pkg/cvo/sync_worker.go

```diff
@@ -617,6 +617,19 @@ func (w *SyncWorker) syncOnce(ctx context.Context, work *SyncWork, maxWorkers in
 				Verified:    info.Verified,
 			})
 			return err
+		} else if !equalDigest(payloadUpdate.Release.Image, work.Desired.Image) {
+			err = fmt.Errorf("release image version %s unchanged, but the image changes from %s to %s", work.Desired.Version, work.Desired.Image, payloadUpdate.Release.Image)
```

Comment:
How does a user fix this problem?

Reply:
they can `oc adm upgrade --clear` (or whatever the equivalent console command is).  Should I say that in the error message?

Reply:
but this will never happen to users, because we are never going to cut two releases with the same name.